cli/cmd/generate.go

```diff
@@ -384,6 +396,32 @@ func addWorkloadOwnerKeyToManifest(manifst *manifest.Manifest, keyPath string) e
 	return nil
 }
 
+func addSeedshareOwnerKeyToManifest(manifst *manifest.Manifest, keyPath string) error {
```

Comment:
Since the function does not need the manifest as input, it might be better to return a `HexString, error` here and inline the `append`.

Reply:
wanted to keep it close to `addWorkloadOwnerKeyToManifest`. I've added a check if the key is already present, so now the manifest is used. ;)

Reply:
We might want to allow duplicate seedshare owner keys when we eventually split the seed.